web/docs/guides/auth/auth-vonage.mdx

```diff
@@ -225,7 +225,7 @@ The second step is the same as the previous section, you need to collect the 6-d
 <Tabs
 defaultValue="JS"
 values={[
-  {label: 'Javascript', value: 'JS'},
+  {label: '', value: 'JS'},
```

Comment:
it seems that this Label has been removed in a few of the tabs - we definitely need it!

Reply:
My bad!